init/init.c

```diff
@@ -152,8 +152,6 @@ static int chroot_luks()
 		waitpid(pid, &wstatus, 0);
 	}
 
-	memset(pass, 0, pass_len);
-
```

Comment:
This `memset()` is here to clean up `pass` from memory, which sits in a region allocated by `malloc` above. Could you please explain further why is not needed?

Reply:
Ah, I edited my `init.c` to have `pass` come from read-only memory (i.e. I just used a string literal instead of taking `KRUN_PASS` from the environment). I must've forgotten to remove this commit with some of the other I used for testing. Will drop this commit on next push.